src/ocean/util/encode/Base64.d

```diff
@@ -553,6 +553,19 @@ unittest
         test!("==")(result, decode("TWE="));
         test!("!=")(result, decode(""));
     }
+
+    // don't decode short things at the end
+    {
+        auto result = cast(Const!(ubyte)[])"Maq";
+        // B should be ignored since it's missing the second byte
+        test!("==")(result, decode("TWFxB"));
```

Comment:
This could use a bit more explanation: What is a *short thing*: Less than two extra bytes?
Maybe make it obvious in code what the valid and what the ignored invalid base 64 bytes are, like
```d
auto plain = cast(Const!(ubyte)[])"Maq";
auto base64 = "TWFx";
// Append one extra byte, say 'B', to base64, which should
// be ignored because it is {insert explanation of "short thing" here}.
test!("==")(plain, decode(encoded ~ "B"));
```